src/Shell/Task/ModelTask.php

```diff
@@ -875,6 +875,16 @@ public function listUnskipped()
     }
 
     /**
+     * Models never have routing prefixes applied.
+     *
+     * @return string
+     */
+    protected function _getPrefix()
```

Comment:
My mistake, I didn't consider the `bake all` context. I was only considering that `bake model` and `bake controller` were separate commands with non-overlapping argument sets. I'm not sure we want `bake all` to expose every option each subcommand exposes. I feel like that could end up making bake all overwhelming to use and complicated to maintain.